tests/cli/component-model/gated-tags.wast

```diff
@@ -0,0 +1,18 @@
+;; RUN: wast --features=-exceptions %
```

Comment:
All of our existing `*.wast` tests also do `--assert default` with `--snapshot ...`, but personally I find it kind of a pain to set that all up and manage it, so my thinking is that it's reasonable to have one-off tests like this which just run the directives themselves.